packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -1413,6 +1413,9 @@ export class AppStudioPluginImpl {
     if (tabDomain) {
       validDomains.push(tabDomain);
     }
+    if (tabEndpoint && isLocalDebug) {
```

Comment:
Do we need to remove `localhost` if adding `localhost:port`?
Or any side-effect if keep both `localhost` and `localhost:port`?

Reply:
From my practice, there is no side-effect to keep both. The [manifest schema doc](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#validdomains) does not have any useful information on the format of host:port...